test/system/200-pod.bats

```diff
@@ -339,6 +339,37 @@ EOF
     run_podman rmi $infra_image
 }
 
+@test "podman pod create should write its pod id to the file specified by --out" {
+    filename=$(mktemp -p ${BATS_TEST_TMPDIR} stdoutXXXXXXXX)
+    local infra_image="k8s.gcr.io/pause:3.5"
+    local pod_name="$(random_string 10 | tr A-Z a-z)"
+
+    # copypasta
+    run_podman --out $filename pod create --name $pod_name --infra-name "$infra_name" --infra-image "$infra_image"
+    is "$output" "" "output from pod create should be empty"
+    ! read -d '' contents <"$filename"
+
+    # next we need to get the pod id from the name that was used
+    run_podman pod inspect $pod_name --format '{{.Id}}'
+    is "$output" "$contents" "output should the pod id that was emitted during creation"
+
+    # enumerate the pods and find the one matching our name
+    run_podman --out $filename pod ls --format json
+    is "$output" "" "output from pod ls should be empty"
+
+    # use the output file to select an entry matching the desired id..
+    jq -r --arg id $contents '.[] | select(.Id == $id) | .Name'.
+    is "$output" "$pod_name" "pod name for the created pod with the retrieved id"
+
+    run_podman --out $filename pod rm -f $pod_name
+    is "$output" "" "output from pod rm should be empty"
+    ! read -d '' contents <"$filename"
```

Comment:
Please don't ever do this. Although this is a common pattern in BATS tests, it makes for unreadable, undebuggable test failures. A possible alternative is to delete this line, and replace the next line (367) with
```bash
    is "$(<$filename)" "$pod_name" ".....etc"
```
Also please note the order of arguments. This (above) is the correct order: `is WHAT-WE-GOT EXPECTED`.